demos/projects/ESPRESSIF/esp32/main/Kconfig.projbuild

```diff
@@ -0,0 +1,84 @@
+menu "Azure IoT middleware for FreeRTOS Sample Configuration"
+
+    config EXAMPLE_WIFI_SSID
```

Comment:
>EXAMPLE [](http://example.com/codeflow?start=11&length=7)

can we prefix macros with something else? EXAMPLE is too generic.
SAMPLE_IOT?

Reply:
That's a good suggestion. Done.